jgiven-spring-junit5/build.gradle

```diff
@@ -1,22 +1,22 @@
 description = "Module for using Spring dependency injection together with JGiven and JUnit 5"
 
 dependencies {
-    compile project(':jgiven-spring')
-    compile project(':jgiven-junit5')
+    implementation project(':jgiven-spring') //TODO: might be api
+    implementation project(':jgiven-junit5') //TODO: might be api
```

Comment:
As far as I know, api more or less replaces compile, so maybe we should use api here?